es6-modules/src/components/todo-app.js

```diff
@@ -0,0 +1,54 @@
+import createElement from '../lib/create-element';
+import noop from '../lib/noop';
```

Comment:
After editing `lib/index.js` you can combine these into a single import statement:
```JAVASCRIPT
import { createElement, noop } from '../lib';
```